README.md

```diff
@@ -6,6 +6,26 @@ The standard object structure is given at https://docs.google.com/document/d/1Jm
 
 The yaml style can be reproduced with `js-yaml` (https://www.npmjs.com/package/js-yaml) using the dump options `{ sortKeys: true, lineWidth: Infinity }`.
 
+## Special Markdown Syntax
+
+Anywhere `markdown` is allowed in a yaml, the following syntax is allowed:
+```md
+[displayed link text](source:file/key)
```

Comment:
I'd like to suggest changing the link format to this: `[displayed link text]($(source:)?file(/key)?)`

That should make the job of parsing out which links are 2D links specifically a bit easier for us.

Reply:
As opposed to hardcoded links, like [this one](https://www.youtube.com/watch?v=dQw4w9WgXcQ)? That's a good point